### DISCUSSION TITLE
could not create a primitive

### DISCUSSION THREAD

**Discussion by cxiaowen** [PaddlePaddle/PaddleOCR#13510] [Q&A]
#### 问题描述 / Problem Description
批量识别代码时出现could not create a primitive这个错误，这个是什么原因？好几个版本

#### 运行环境 / Runtime Environment
- OS:windows
- Paddle:
- PaddleOCR:

**Comment by jingsongliujing** [comment]
你是不是开启MKL加速了，把MKL加速关闭试试？

**Comment by cxiaowen** [comment]
ocr = PaddleOCR(
            use_angle_cls=True,
            det_db_unclip_ratio=2.0,  # 调整为适合的值
            det_db_box_thresh=0.3,    # 调整为适合的值
            det_limit_side_len=4000,  # 提高分辨率限制
            det_algorithm='DB',       # 尝试不同的检测算法，例如'EAST'
            rec_algorithm='SVTR_LCNet',
            enable_mkldnn=False,
            use_gpu=True
        )是这个，已经关了，还是会

**Comment by cxiaowen** [comment]
> 你是不是开启MKL加速了，把MKL加速关闭试试？

  ocr = PaddleOCR(
        use_angle_cls=True,
        det_db_unclip_ratio=2.0,  # 调整为适合的值
        det_db_box_thresh=0.3,    # 调整为适合的值
        det_limit_side_len=4000,  # 提高分辨率限制
        det_algorithm='DB',       # 尝试不同的检测算法，例如'EAST'
        rec_algorithm='SVTR_LCNet',
        enable_mkldnn=False,
        use_gpu=True
    )是这个，已经关了，还是会发生这种情况

**Comment by GreatV** [comment]
simliar issues:
- https://github.com/PaddlePaddle/PaddleOCR/issues/10721
- https://github.com/PaddlePaddle/PaddleOCR/issues/13407

**Comment by GreatV** [comment]
可以把你的问题反馈到
- https://github.com/PaddlePaddle/PaddleOCR/issues/13407

**Comment by GreatV** [comment]
还有你的运行环境是什么，用的paddleocr，paddle版本是什么，cuda版本是什么，cpu是什么型号的

**Comment by cxiaowen** [comment]
paddle-bfloat                            0.1.7
paddleclas                               2.5.2
paddleocr                                2.7.3
paddlepaddle                             2.6.1

> 还有你的运行环境是什么，用的paddleocr，paddle版本是什么，cuda版本是什么，cpu是什么型号的
paddle-bfloat                            0.1.7
paddleclas                               2.5.2
paddleocr                                2.7.3
paddlepaddle                             2.6.1   cuda11.7

**Comment by IvenTao** [comment]
我也碰到了这个问题，兄弟你现在解决这个问题了吗
